price-feeder/oracle/provider/osmosisv2.go

```diff
@@ -18,8 +18,8 @@ import (
 const (
 	osmosisV2WSHost   = "api.osmo-api.network.umee.cc"
 	osmosisV2WSPath   = "ws"
-	osmosisV2RestHost = "https://api-osmosis.imperator.co"
-	osmosisV2RestPath = "/pairs/v1/summary"
+	osmosisV2RestHost = "https://api.osmo-api.network.umee.cc"
+	osmosisV2RestPath = "/assetpairs"
```

Comment:
why we are not using the official osmosis node?

Reply:
See the osmosis-api limitations bullet point in #792. We've provisioned our own osmosis websocket api to get around these limitations (https://github.com/umee-network/osmosis-api).